backend/src/test/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapperTest.java

```diff
@@ -285,11 +282,11 @@ void map_WhenReportedSammToProductItemStock_ReturnsMultipleReportedProductItemSt
         mpr.setPartnerBuysMaterial(false);
         mpr.setPartnerSuppliesMaterial(true);
         mpr.setPartnerMaterialNumber(SUPPLIER_MAT_NUMBER);
+        mpr.setPartnerCXNumber(CX_MAT_NUMBER);
```

Comment:
ReportedProductStocks are received as a customer partner from a supplier partner. From my understanding in these cases the material.materialNumberCx should be different to the mpr.partnerCXNumber:
- When creating a material, I can define my own materialNumberCx. 
- When receiving an PartTypeInformation (I'm acting as customer and will receive it from masterdata.logic.aadapter.PartTypeInformationSammMapper.updateMaterialFromSamm), I write the suppliers number into the mpr.

Please enhance test cases accordingly to differentiate by Supplier CX_MAT_NUMBER and Customer CX_MAT_NUMBER as this may reveal further problems. Same issue persists in DataInjectionCommandLineRunner.

Reply:
Since we are facing the case here, that the mpr, which is created in line 281, is geared to be from the Supplier's perspective, there should not be a PartnerCXNumber in that mpr (or at least: it should not matter, if it does not exist)

Reply:
> Please enhance test cases accordingly to differentiate by Supplier CX_MAT_NUMBER and Customer CX_MAT_NUMBER as this may reveal further problems. Same issue persists in DataInjectionCommandLineRunner.

After @eschrewe comment, is this still relevant? @tom-rm-meyer-ISST

Reply:
alright, I just want to have it in a uniform way. @f-zimmer could you please add some documentation in the fields on the mpr and the material how they are filled?